html/css/app.css

```diff
@@ -948,3 +940,15 @@ ul ul ul {
   position: sticky;
   top: 50%;
 }
+
+.plus-icon {
```

Comment:
This could be named `icon-overlay-top-right` to make it reusable. Can always change it the next time it's needed as well.